src/main/resources/db/migration/V1_1__add_shedlock_table.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE shedlock (
```

Comment:
Makes so much sense that this needs a table. Had thought it was voodoo magic before, but I guess this library essentially proxies the concurrency issue to the database we use